src/domains/User/UserProfile/UserProfile.tsx

```diff
@@ -19,37 +31,129 @@ const useStyles = makeStyles((theme) => ({
+                        <Form onSubmit={handleSubmit(sendName)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'First Name' }}
+                                    label='First Name'
+                                    aria-label='First Name'
+                                    id='fName'
+                                    required
+                                    type='text'
+                                    value={regForm.firstName}
+                                    onChange={handleChange('firstName')}
+                                />
+                            </FormContent>
+                        </Form>
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>
+                        <Form onSubmit={handleSubmit(sendName)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'Last Name' }}
+                                    label='Last Name'
+                                    aria-label='Last Name'
+                                    id='lName'
+                                    required
+                                    type='text'
+                                    value={regForm.lastName}
+                                    onChange={handleChange('lastName')}
+                                />
+                            </FormContent>
+                        </Form>
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>
+                        <Form onSubmit={handleSubmit(sendEmail)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'E-mail' }}
+                                    label='Email'
+                                    aria-label='E-mail'
+                                    id='email'
+                                    required
+                                    type='email'
+                                    value={regForm.email}
+                                    onChange={handleChange('email')}
+                                />
+                            </FormContent>
+                        </Form>
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>
+                        <h3>Reset Password</h3>
+                        <PasswordResetForm onSuccess={() => {}} token={user?._id || 'undefined'} />
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>
+                        <Divider />
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>
+                        <h3>Logout</h3>
+                        <Button
+                            href='https://prytaneum.io/logout'
```

Comment:
Remember to make these links relative to the app itself.  Or you could just make this a `// TODO:`.  We're migrating the project a bit, so this may be unnecessary work at the moment.